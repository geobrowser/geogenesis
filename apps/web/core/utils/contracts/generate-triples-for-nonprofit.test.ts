import { describe, expect, it } from 'vitest';

import { generateTriplesForNonprofit } from './generate-triples-for-nonprofit';

describe('Generating triples in a nonprofit space', () => {
  it('generates initial content template', () => {
    expect(generateTriplesForNonprofit('spaceConfigEntityId', 'spaceName', 'spaceAddress')).toMatchSnapshot();
  });
});
