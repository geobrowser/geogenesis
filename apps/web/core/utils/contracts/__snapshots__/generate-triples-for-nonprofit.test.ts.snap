// Vitest Snapshot v1

exports[`Generating triples in a nonprofit space > generates initial content template 1`] = `
[
  {
    "attributeId": "be745973-05a9-4cd0-a46d-1c5538270faf",
    "attributeName": "Foreign Types",
    "entityId": "spaceConfigEntityId",
    "entityName": "spaceName",
    "id": "spaceAddress:spaceConfigEntityId:be745973-05a9-4cd0-a46d-1c5538270faf:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
  {
    "attributeId": "be745973-05a9-4cd0-a46d-1c5538270faf",
    "attributeName": "Foreign Types",
    "entityId": "spaceConfigEntityId",
    "entityName": "spaceName",
    "id": "spaceAddress:spaceConfigEntityId:be745973-05a9-4cd0-a46d-1c5538270faf:3d31f766-b651-48af-a357-271343a773de",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "3d31f766-b651-48af-a357-271343a773de",
      "name": "Tag",
      "type": "entity",
    },
  },
  {
    "attributeId": "be745973-05a9-4cd0-a46d-1c5538270faf",
    "attributeName": "Foreign Types",
    "entityId": "spaceConfigEntityId",
    "entityName": "spaceName",
    "id": "spaceAddress:spaceConfigEntityId:be745973-05a9-4cd0-a46d-1c5538270faf:682fbeff-41e2-42cd-a7f9-c4909136a8c5",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "682fbeff-41e2-42cd-a7f9-c4909136a8c5",
      "name": "Post",
      "type": "entity",
    },
  },
  {
    "attributeId": "be745973-05a9-4cd0-a46d-1c5538270faf",
    "attributeName": "Foreign Types",
    "entityId": "spaceConfigEntityId",
    "entityName": "spaceName",
    "id": "spaceAddress:spaceConfigEntityId:be745973-05a9-4cd0-a46d-1c5538270faf:cb9d261d-456b-4eaf-87e5-1e9faa441867",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "cb9d261d-456b-4eaf-87e5-1e9faa441867",
      "name": "Project",
      "type": "entity",
    },
  },
  {
    "attributeId": "be745973-05a9-4cd0-a46d-1c5538270faf",
    "attributeName": "Foreign Types",
    "entityId": "spaceConfigEntityId",
    "entityName": "spaceName",
    "id": "spaceAddress:spaceConfigEntityId:be745973-05a9-4cd0-a46d-1c5538270faf:ce59ccc1-2ac5-4ace-8f82-09322434733d",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "ce59ccc1-2ac5-4ace-8f82-09322434733d",
      "name": "Finance Summary",
      "type": "entity",
    },
  },
  {
    "attributeId": "577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac",
    "attributeName": "Row Type",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Pages",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
  {
    "attributeId": "b0f2d71a-79ca-4dc4-9218-e3e40dfed103",
    "attributeName": "Filter",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Pages",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:b0f2d71a-79ca-4dc4-9218-e3e40dfed103:5853e24b-99e9-4ac8-9f59-f8d4f89b8a7a",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "5853e24b-99e9-4ac8-9f59-f8d4f89b8a7a",
      "type": "string",
      "value": "{typeIds_contains_nocase: [\\"1a9fc4a0-0fec-4eea-a075-eec7ebd0d043\\"], entityOf_: {space: \\"spaceAddress\\"}}",
    },
  },
  {
    "attributeId": "dd4999b9-77f0-4c2b-a02b-5a26b233854e",
    "attributeName": "Parent Entity",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Pages",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:dd4999b9-77f0-4c2b-a02b-5a26b233854e:spaceConfigEntityId",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "spaceConfigEntityId",
      "name": "spaceName",
      "type": "entity",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Pages",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:name:79ffc731-1432-417c-abf8-fba980e38599",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "79ffc731-1432-417c-abf8-fba980e38599",
      "type": "string",
      "value": "Pages",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Pages",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:type:88d59252-17ae-4d9a-a367-24710129eb47",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "88d59252-17ae-4d9a-a367-24710129eb47",
      "name": "Table Block",
      "type": "entity",
    },
  },
  {
    "attributeId": "beaba5cb-a677-41a8-b353-77030613fc70",
    "attributeName": "Blocks",
    "entityId": "spaceConfigEntityId",
    "entityName": "spaceName",
    "id": "spaceAddress:spaceConfigEntityId:beaba5cb-a677-41a8-b353-77030613fc70:a9155e60-b2ca-4e95-b216-1daaf5d7c206",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "a9155e60-b2ca-4e95-b216-1daaf5d7c206",
      "type": "string",
      "value": "[\\"efbf7254-9b39-4219-9c2b-16aaa02de50a\\"]",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "767f5eee-7290-4869-83d3-671d76640562",
    "entityName": "Posts",
    "id": "spaceAddress:767f5eee-7290-4869-83d3-671d76640562:name:7b0e46ad-e6f0-410e-a527-470b1f8f00ae",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "7b0e46ad-e6f0-410e-a527-470b1f8f00ae",
      "type": "string",
      "value": "Posts",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "767f5eee-7290-4869-83d3-671d76640562",
    "entityName": "Posts",
    "id": "spaceAddress:767f5eee-7290-4869-83d3-671d76640562:type:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
  {
    "attributeId": "577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac",
    "attributeName": "Row Type",
    "entityId": "58995087-6405-4d29-a0fe-2ab4921d32cd",
    "entityName": "Posts",
    "id": "spaceAddress:58995087-6405-4d29-a0fe-2ab4921d32cd:577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac:682fbeff-41e2-42cd-a7f9-c4909136a8c5",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "682fbeff-41e2-42cd-a7f9-c4909136a8c5",
      "name": "Post",
      "type": "entity",
    },
  },
  {
    "attributeId": "b0f2d71a-79ca-4dc4-9218-e3e40dfed103",
    "attributeName": "Filter",
    "entityId": "58995087-6405-4d29-a0fe-2ab4921d32cd",
    "entityName": "Posts",
    "id": "spaceAddress:58995087-6405-4d29-a0fe-2ab4921d32cd:b0f2d71a-79ca-4dc4-9218-e3e40dfed103:c59d8a4f-9669-417b-859b-100c674c78db",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "c59d8a4f-9669-417b-859b-100c674c78db",
      "type": "string",
      "value": "{typeIds_contains_nocase: [\\"682fbeff-41e2-42cd-a7f9-c4909136a8c5\\"], entityOf_: {space: \\"spaceAddress\\"}}",
    },
  },
  {
    "attributeId": "dd4999b9-77f0-4c2b-a02b-5a26b233854e",
    "attributeName": "Parent Entity",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Posts",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:dd4999b9-77f0-4c2b-a02b-5a26b233854e:767f5eee-7290-4869-83d3-671d76640562",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "767f5eee-7290-4869-83d3-671d76640562",
      "name": "spaceName",
      "type": "entity",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "58995087-6405-4d29-a0fe-2ab4921d32cd",
    "entityName": "Posts",
    "id": "spaceAddress:58995087-6405-4d29-a0fe-2ab4921d32cd:name:8e150f52-b8a5-4722-b4cf-c035b52d9430",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "8e150f52-b8a5-4722-b4cf-c035b52d9430",
      "type": "string",
      "value": "Posts",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Posts",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:type:88d59252-17ae-4d9a-a367-24710129eb47",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "88d59252-17ae-4d9a-a367-24710129eb47",
      "name": "Table Block",
      "type": "entity",
    },
  },
  {
    "attributeId": "beaba5cb-a677-41a8-b353-77030613fc70",
    "attributeName": "Blocks",
    "entityId": "767f5eee-7290-4869-83d3-671d76640562",
    "entityName": "spaceName",
    "id": "spaceAddress:767f5eee-7290-4869-83d3-671d76640562:beaba5cb-a677-41a8-b353-77030613fc70:ccaa4be0-b963-403a-bc87-f51604882376",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "ccaa4be0-b963-403a-bc87-f51604882376",
      "type": "string",
      "value": "[\\"58995087-6405-4d29-a0fe-2ab4921d32cd\\"]",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "5088e94f-03f1-4045-8bb9-36a7821d0717",
    "entityName": "Finances",
    "id": "spaceAddress:5088e94f-03f1-4045-8bb9-36a7821d0717:name:9329d2f0-9517-460f-9ca9-67dc9da7ccc8",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "9329d2f0-9517-460f-9ca9-67dc9da7ccc8",
      "type": "string",
      "value": "Finances",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "5088e94f-03f1-4045-8bb9-36a7821d0717",
    "entityName": "Finances",
    "id": "spaceAddress:5088e94f-03f1-4045-8bb9-36a7821d0717:type:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:0b623cb1-ba1d-407a-b9cc-58becb52b531",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "0b623cb1-ba1d-407a-b9cc-58becb52b531",
      "name": "Other revenue sources",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:15645106-d203-4db1-8692-012a6e06349f",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "15645106-d203-4db1-8692-012a6e06349f",
      "name": "Non-financial assets",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:5572bb78-1e9b-4752-8333-72e7c2c90c8b",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "5572bb78-1e9b-4752-8333-72e7c2c90c8b",
      "name": "Construction costs",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:6c90a975-5da7-4f5a-9401-514d30dd9926",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "6c90a975-5da7-4f5a-9401-514d30dd9926",
      "name": "Other expenses",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:73984ff3-11e6-4b42-bd09-07f34d4be0e2",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "73984ff3-11e6-4b42-bd09-07f34d4be0e2",
      "name": "Grants",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:8a573366-1d86-48b6-b861-c48aad11486f",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "8a573366-1d86-48b6-b861-c48aad11486f",
      "name": "Program services fees",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:9e94fc0c-2f6e-4fd6-894b-eef6bfe64d49",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "9e94fc0c-2f6e-4fd6-894b-eef6bfe64d49",
      "name": "Contributions",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:bc6d8191-1626-42b2-a8fc-6d639535306c",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "bc6d8191-1626-42b2-a8fc-6d639535306c",
      "name": "Salaries + Benefits",
      "type": "entity",
    },
  },
  {
    "attributeId": "388ad59b-1cc7-413c-a0bb-34a4de48c758",
    "attributeName": "Shown Columns",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:388ad59b-1cc7-413c-a0bb-34a4de48c758:e769b6ac-2387-4b37-919b-9cab193868fd",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "e769b6ac-2387-4b37-919b-9cab193868fd",
      "name": "Total revenue",
      "type": "entity",
    },
  },
  {
    "attributeId": "577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac",
    "attributeName": "Row Type",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac:ce59ccc1-2ac5-4ace-8f82-09322434733d",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "ce59ccc1-2ac5-4ace-8f82-09322434733d",
      "name": "Finance Summary",
      "type": "entity",
    },
  },
  {
    "attributeId": "b0f2d71a-79ca-4dc4-9218-e3e40dfed103",
    "attributeName": "Filter",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:b0f2d71a-79ca-4dc4-9218-e3e40dfed103:7c3fb5f2-1c20-46dc-9b93-ee5740ad33e6",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "7c3fb5f2-1c20-46dc-9b93-ee5740ad33e6",
      "type": "string",
      "value": "{typeIds_contains_nocase: [\\"682fbeff-41e2-42cd-a7f9-c4909136a8c5\\"], entityOf_: {space: \\"spaceAddress\\"}}",
    },
  },
  {
    "attributeId": "dd4999b9-77f0-4c2b-a02b-5a26b233854e",
    "attributeName": "Parent Entity",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:dd4999b9-77f0-4c2b-a02b-5a26b233854e:5088e94f-03f1-4045-8bb9-36a7821d0717",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "5088e94f-03f1-4045-8bb9-36a7821d0717",
      "name": "Finances",
      "type": "entity",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:name:ceda606c-d89c-42b3-bcbe-bcb09351b3e7",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "ceda606c-d89c-42b3-bcbe-bcb09351b3e7",
      "type": "string",
      "value": "Finance Summary",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "b741e80b-2b84-4a54-9248-41871a119de1",
    "entityName": "Finance Summaries",
    "id": "spaceAddress:b741e80b-2b84-4a54-9248-41871a119de1:type:88d59252-17ae-4d9a-a367-24710129eb47",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "88d59252-17ae-4d9a-a367-24710129eb47",
      "name": "Table Block",
      "type": "entity",
    },
  },
  {
    "attributeId": "beaba5cb-a677-41a8-b353-77030613fc70",
    "attributeName": "Blocks",
    "entityId": "5088e94f-03f1-4045-8bb9-36a7821d0717",
    "entityName": "spaceName",
    "id": "spaceAddress:5088e94f-03f1-4045-8bb9-36a7821d0717:beaba5cb-a677-41a8-b353-77030613fc70:bfe9122d-e6b9-4e06-a937-44ec74c18392",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "bfe9122d-e6b9-4e06-a937-44ec74c18392",
      "type": "string",
      "value": "[\\"b741e80b-2b84-4a54-9248-41871a119de1\\"]",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "e60016c7-2b77-42da-a038-c5c5778b8ba4",
    "entityName": "Projects",
    "id": "spaceAddress:e60016c7-2b77-42da-a038-c5c5778b8ba4:name:f3201cb9-8489-4a9c-bba0-831afdbc3cff",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "f3201cb9-8489-4a9c-bba0-831afdbc3cff",
      "type": "string",
      "value": "Projects",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "e60016c7-2b77-42da-a038-c5c5778b8ba4",
    "entityName": "Projects",
    "id": "spaceAddress:e60016c7-2b77-42da-a038-c5c5778b8ba4:type:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
  {
    "attributeId": "577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac",
    "attributeName": "Row Type",
    "entityId": "4d956700-590b-4da7-986f-8ae48d5cfd1a",
    "entityName": "Projects",
    "id": "spaceAddress:4d956700-590b-4da7-986f-8ae48d5cfd1a:577bd9fb-b29e-4e2b-b5f8-f48aedbd26ac:682fbeff-41e2-42cd-a7f9-c4909136a8c5",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "682fbeff-41e2-42cd-a7f9-c4909136a8c5",
      "name": "Project",
      "type": "entity",
    },
  },
  {
    "attributeId": "b0f2d71a-79ca-4dc4-9218-e3e40dfed103",
    "attributeName": "Filter",
    "entityId": "4d956700-590b-4da7-986f-8ae48d5cfd1a",
    "entityName": "Projects",
    "id": "spaceAddress:4d956700-590b-4da7-986f-8ae48d5cfd1a:b0f2d71a-79ca-4dc4-9218-e3e40dfed103:12e647db-01dd-4a50-9c66-56cb464bc06a",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "12e647db-01dd-4a50-9c66-56cb464bc06a",
      "type": "string",
      "value": "{typeIds_contains_nocase: [\\"682fbeff-41e2-42cd-a7f9-c4909136a8c5\\"], entityOf_: {space: \\"spaceAddress\\"}}",
    },
  },
  {
    "attributeId": "dd4999b9-77f0-4c2b-a02b-5a26b233854e",
    "attributeName": "Parent Entity",
    "entityId": "4d956700-590b-4da7-986f-8ae48d5cfd1a",
    "entityName": "Projects",
    "id": "spaceAddress:4d956700-590b-4da7-986f-8ae48d5cfd1a:dd4999b9-77f0-4c2b-a02b-5a26b233854e:e60016c7-2b77-42da-a038-c5c5778b8ba4",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "e60016c7-2b77-42da-a038-c5c5778b8ba4",
      "name": "Projects",
      "type": "entity",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "4d956700-590b-4da7-986f-8ae48d5cfd1a",
    "entityName": "Projects",
    "id": "spaceAddress:4d956700-590b-4da7-986f-8ae48d5cfd1a:name:6b31665f-1394-4a57-99ef-4787bed3c3fe",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "6b31665f-1394-4a57-99ef-4787bed3c3fe",
      "type": "string",
      "value": "Projects",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "efbf7254-9b39-4219-9c2b-16aaa02de50a",
    "entityName": "Projects",
    "id": "spaceAddress:efbf7254-9b39-4219-9c2b-16aaa02de50a:type:88d59252-17ae-4d9a-a367-24710129eb47",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "88d59252-17ae-4d9a-a367-24710129eb47",
      "name": "Table Block",
      "type": "entity",
    },
  },
  {
    "attributeId": "beaba5cb-a677-41a8-b353-77030613fc70",
    "attributeName": "Blocks",
    "entityId": "e60016c7-2b77-42da-a038-c5c5778b8ba4",
    "entityName": "spaceName",
    "id": "spaceAddress:e60016c7-2b77-42da-a038-c5c5778b8ba4:beaba5cb-a677-41a8-b353-77030613fc70:9ac0f3aa-dfa6-422c-9e85-40b0ef6ada00",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "9ac0f3aa-dfa6-422c-9e85-40b0ef6ada00",
      "type": "string",
      "value": "[\\"4d956700-590b-4da7-986f-8ae48d5cfd1a\\"]",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "aaa9ebf3-fef0-498e-8b81-f5d237fc5210",
    "entityName": "About Us",
    "id": "spaceAddress:aaa9ebf3-fef0-498e-8b81-f5d237fc5210:name:3cf913f7-75a1-44ab-b572-8dd8aaeedd71",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "3cf913f7-75a1-44ab-b572-8dd8aaeedd71",
      "type": "string",
      "value": "About Us",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "aaa9ebf3-fef0-498e-8b81-f5d237fc5210",
    "entityName": "About Us",
    "id": "spaceAddress:aaa9ebf3-fef0-498e-8b81-f5d237fc5210:type:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
  {
    "attributeId": "name",
    "attributeName": "Name",
    "entityId": "dc51a029-005f-4975-8350-b845da28fc22",
    "entityName": "Get Involved",
    "id": "spaceAddress:dc51a029-005f-4975-8350-b845da28fc22:name:b4ab6d00-64a9-42bb-bca3-980961d1a19b",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "b4ab6d00-64a9-42bb-bca3-980961d1a19b",
      "type": "string",
      "value": "Get Involved",
    },
  },
  {
    "attributeId": "type",
    "attributeName": "Types",
    "entityId": "dc51a029-005f-4975-8350-b845da28fc22",
    "entityName": "Get Involved",
    "id": "spaceAddress:dc51a029-005f-4975-8350-b845da28fc22:type:1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
    "space": "spaceAddress",
    "type": "createTriple",
    "value": {
      "id": "1a9fc4a0-0fec-4eea-a075-eec7ebd0d043",
      "name": "Page",
      "type": "entity",
    },
  },
]
`;
